index.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
```

Comment:
Wow very short and simple code 
I like 👍

Reply:
Thank you! Although, I didn't finish the exercise :)